docs/database/database-spans.md

```diff
@@ -76,6 +77,17 @@ and SHOULD adhere to one of the following values, provided they are accessible:
 If a corresponding `{target}` value is not available for a specific operation, the instrumentation SHOULD omit the `{target}`.
 For example, for an operation describing SQL query on an anonymous table like `SELECT * FROM (SELECT * FROM table) t`, span name should be `SELECT`.
 
+## Sanitization of `db.query.text`
```

Comment:
Can it be moved down to be under the attribute table? Then we'll first introduce the attribute and then talk about its sanitization.

Reply:
Makes sense, moved this part below.